src/widgets/boundary.tsx

```diff
@@ -48,73 +52,77 @@ class ErrorBoundary extends React.Component<ErrorBoundaryProps, ErrorBoundarySta
           radius="lg"
           shadow="sm"
           withBorder
+          h="calc(100% - 20px)"
         >
-          <Center>
-            <Stack align="center">
-              <IconBug color="white" />
-              <Stack spacing={0} align="center">
-                <Title order={4} color="white" align="center">
-                  {this.props.t('card.title')}
-                </Title>
-                {this.state.error && (
-                  <Text color="white" align="center" size="sm">
-                    {this.state.error.toString()}
-                  </Text>
-                )}
+          <WidgetsMenu integration={this.props.integration} widget={this.props.widget} />
+          <ScrollArea h="100%" type="auto" offsetScrollbars>
+            <Center>
+              <Stack align="center" spacing="xs">
+                <IconBug color="white" />
+                <Stack spacing={0} align="center">
+                  <Title order={5} color="white" align="center">
+                    {this.props.t('card.title')}
+                  </Title>
+                  {this.state.error && (
+                    <Text color="white" align="center" size="sm">
+                      {this.state.error.toString()}
+                    </Text>
+                  )}
+                </Stack>
+                <Group spacing="xs">
+                  <Button
+                    onClick={() =>
+                      openModal({
+                        title: 'Your widget had an error',
+                        children: (
+                          <>
+                            <Text size="sm" mb="sm">
+                              {this.props.t('modal.text')}
+                            </Text>
+                            {this.state.error && (
+                              <>
+                                <Text weight="bold" size="sm">
+                                  {this.props.t('modal.label')}
+                                </Text>
+                                <Code block>{this.state.error.toString()}</Code>
+                              </>
+                            )}
+                            <Button
+                              sx={(theme) => ({
+                                backgroundColor: theme.colors.gray[8],
```

Comment:
Maybe you could use the property `bg` for the default background color

Reply:
Yes, but it's not really worth it when we have to set the hover action anyway.